scripts/commands/getlocalvars.lua

```diff
@@ -0,0 +1,33 @@
+-----------------------------------
+-- Func: getlocalvars
+-- Desc: Gets all local vars of a target
+-----------------------------------
+local commandObj = {}
+
+commandObj.cmdprops =
+{
+    permission = 3,
```

Comment:
Except for extremely dangerous actions, we normally start at `1` and let servers adjust them if they need more restriction

This doesn't seem dangerous.

Reply:
> Except for extremely dangerous actions, we normally start at `1` and let servers adjust them if they need more restriction
> 
> This doesn't seem dangerous.

Ah, ok, thanks for info, yeah I think not very dangerous in itself. I adjusted the script to permission 1.